syntax = "proto3";

// 准备发送的数据
message RequestMsg
{
    // 秘钥协商 ---> 秘钥校验 ---> 秘钥注销
    int32 cmdType = 1;  // 报文类型
    bytes clientID = 2; // 客户端编号
    bytes serverID = 3; // 服务端编号
    bytes sign = 4;
    bytes data = 5;
}

message RespondMsg 
{
    bool status = 1;    // 返回值
    int32 seckeyID = 2; // 对称秘钥编号keysn
    bytes clientID = 3; // 客户端编号
    bytes serverID = 4; // 服务端编号
    bytes data = 5;     //服务器随机数
}
